docs/standard/whats-new/whats-new-in-dotnet-standard.md

```diff
@@ -12,16 +13,15 @@ ms.workload:
   - "dotnet"
   - "dotnetcore"
 ---
-
 # What's new in the .NET Standard
 
-The .NET Standard is a formal specification that defines a versioned set of APIs which must be available on .NET implementations that comply with that version of the standard. The .NET Standard is targeted at library developers. A library that targets a .NET Standard version can be used on any .NET Framework, .NET Core, or Xamarin implementation that supports that version of the standard.
+The .NET Standard is a formal specification that defines a versioned set of APIs that must be available on .NET implementations that comply with that version of the standard. The .NET Standard is targeted at library developers. A library that targets a .NET Standard version can be used on any .NET Framework, .NET Core, or Xamarin implementation that supports that version of the standard.
 
 The most recent version of the .NET Standard is 2.0. It is included with the .NET Core 2.0 SDK, as well as with Visual Studio 2017 Version 15.3 with the .NET Core workload installed.
 
 ## Supported .NET implementations
 
-The .NET Standard 2.0 is supported by the following .NET implementations:
+The .NET Standard 2.0 is supported by the following .NET implementations, including later versions of those:
```

Comment:
nit: those -> these
But I wonder if it might not be clearer to append "or later" to each of the implementations in the list.